dev/Deployment/DeploymentManager.cpp

```diff
@@ -389,6 +474,12 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         std::wstring currentPackageFullName{ packageFullName };
         auto currentPackageInfo{ GetPackageInfoForPackage(currentPackageFullName) };
 
+        // Save whether the current package is of same publisher as framework package's Microsoft pulbisher
```

Comment:
Typo: ...publisher (end of line)

Or even better, cut the special-case and bye bye typos :-)